dev/com.ibm.websphere.appserver.features/visibility/private/io.openliberty.xmlws.common-3.0.feature

```diff
@@ -0,0 +1,64 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
```

Comment:
Created a new common-3.0 feature so that the client and server side features can share dependencies. I had to pull in the dependencies from the `jaxws-2.3` and `jaxrs-2.1`internal `com.ibm.websphere.appserver.cxf.internal-1.0` feature, since we no longer share an implementation with `restfulWs-3.0`